spec2sdk/client/templates/client.j2

```diff
@@ -25,6 +25,20 @@ class APIClient:
     def __init__(self, *, http_client: HTTPClientProtocol):
         self._http_client = http_client
 
+    def serialize[T](self, instance: T, instance_type: Type[T] | TypeAliasType, content_type: str | None) -> bytes:
```

Comment:
```suggestion
    def serialize[T](self, data: T, data_type: Type[T] | TypeAliasType, content_type: str | None) -> bytes:
```
I'd call it "data", because "instance" sounds a lot like class. But the data can be "int" as well.

Reply:
I'm fine with the changes, but `5` is an instance of the `int` class :)

Reply:
You are right :) Sometimes I forget about the beauty of python types.